shared/src/commonMain/kotlin/com/kcc/kmmhackathon/shared/utility/LocationUtil.kt

```diff
@@ -0,0 +1,29 @@
+package com.kcc.kmmhackathon.shared.utility
+
+import kotlin.math.*
+
+typealias Radian = Double
+typealias Degree = Double
+
+class LocationUtil {
+
+    fun getDistance(lat1: Double, lon1: Double,lat2: Double, lon2: Double, inMiles: Boolean): Double {
+
+        // Haversine formula
+        val earthRadius = 3958.75
+        val dLat = (lat1-lat2).toRadian()
+        val dLon = (lon1-lon2).toRadian()
+
+        val a: Double = sin(dLat / 2) * sin(dLat / 2) +
+                cos(lat2.toRadian()) * cos(lat1.toRadian()) *
+                sin(dLon / 2) * sin(dLon / 2);
+        val c: Double = 2 * atan2(sqrt(a), sqrt(1 - a))
+        val distance: Double = earthRadius * c
+
+        return if (inMiles) { distance.round(2) } else { distance.toKilometres().round(2) }
```

Comment:
Is this common in kotlin to have `if`'s and `else` on the same lines?

Reply:
I've never personally done that but not sure if it's a "don't" - @KaneCheshire do you know?

Reply:
There's no ternary operator in Kotlin unfortunately so you have to do `if else` on the same line, but I don't think you need the `{}`

```kotlin
return if (inMiles) distance.round(2) else distance.toKilometers().round(2)
```